mpp/shmem.h.in

```diff
@@ -18,6 +18,10 @@
 
 #include <stddef.h>
 #include <complex.h>
```

Comment:
http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1548.pdf

Page 205, 7.3.1.4:

>  The macro `complex` expands to `_Complex`; the macro `_Complex_I` expands to a constant expression of type `const float _Complex`, with the value of the imaginary unit.

The spec defines float _Complex and double _Complex as types, but _Complex itself is just a keyword.
pg 59, 6.2.5.11:

> There are three complex types, designated as `float _Complex`, `double _Complex`, and `long double _Complex`. (Complex types are a conditional feature that implementations need not support; see 6.10.8.3.) The real floating and complex types are collectively called the floating types.